src/http/server.rs

```diff
@@ -20,6 +21,19 @@ fn graphiql() -> Result<Response<Body>> {
   )))
 }
 
+pub fn add_headers(mut resp: Response<Body>, headers: Option<BTreeMap<String, String>>) -> Result<Response<Body>> {
+  if let Some(headers) = headers {
+    for (k, v) in headers {
+      resp.headers_mut().insert(
+        HeaderName::from_bytes(k.as_bytes())?,
+        HeaderValue::from_bytes(v.as_bytes())?,
+      );
+    }
+  }
```

Comment:
We can convert `Vec<String, String>` directly to a HeaderMap is that's going to be less work and should improve performance slightly.

Reply:
Done